### DISCUSSION TITLE
Feature request:Adding sidebar navigation

### DISCUSSION THREAD

**Discussion by H1Mohamed** [saadeghi/daisyui#2394] [Ideas / feature requests]
one of the elements that's stopping me from using daisyui is the lack of a sidebar navigation menu(along side my hatred for tailwindcss but I can go over that) and as I can see that I\m pretty good in css,its not a bet I'm willing to take to learn tailwindcss + daisyui just to have to build a sidebar by my self,but If you can guys make it....I will joing daisyui after learning tailwindcss

**Comment by saadeghi** [comment]
It's here:  
https://daisyui.com/components/drawer/

**Reply by H1Mohamed** [reply]
I want it a version like the discord and twiter sidebar,alwase visibl on the screen and supports icons

**Comment by saadeghi** [comment]
>alwase visibl on the screen

[`drawer-open` class](https://daisyui.com/components/drawer/#responsive) makes `drawer` visible.

>supports icons  

Here you can find all the examples of menu with icons:  
https://daisyui.com/components/menu/

And if you want the sidebar to change sizes on specific screen or hide menu labels, you can simply use Tailwind responsive prefixes like `lg:` to apply styles for a specific screen.  

Here's an example. Narrow sidebar with icon only on small screens. wide sidebar with icon and text on large screen:  
https://play.tailwindcss.com/gRDI7K0Pai?size=1048x720
